fxa-auth-db-server/index.js

```diff
@@ -117,6 +119,29 @@ function createServer(db) {
 
   api.get('/__heartbeat__', reply(db.ping))
 
+  api.put(
+    '/account/:id/device/:deviceId',
+    function (req, res, next) {
+      db.upsertDevice(req.params.uid, req.params.deviceId, req.body)
```

Comment:
@rfk @dannycoates is there a way to integrate the Activity Events into this? Do we we have a way to tell that a new device was added and log `device.added` to Heka? (could be separate bug)